user/views.py

```diff
@@ -0,0 +1,14 @@
+from django.shortcuts import render
+from django.views.generic.edit import CreateView
+from .forms import Accountform
+from django.views.generic import TemplateView
+
+class Start(TemplateView):
```

Comment:
Viewの命名は `~~View`の形で統一しましょう！
```suggestion
class StartView(TemplateView):
```